tensorflow/tsl/platform/default/env.cc

```diff
@@ -271,16 +271,17 @@ void PosixEnv::GetLocalTempDirectories(std::vector<string>* list) {
     getenv("TMP"),
 
 #if defined(__ANDROID__)
-    "/data/local/tmp",
+    "/data/local/tmp/",
```

Comment:
Remove trailing slash

Reply:
I added them explicitly to speed up computation ;) I can't help myself not optimize code that I modify!

On line 288, the trailing slash will be added. Anyway, no need to spend more time on this. Thanks for the review.

Reply:
:joy: Sorry I removed them again before seeing this message :P You can file another PR if you dare (JUST KIDDING)

Reply:
haha! Should I do ;)